agent/Makefile

```diff
@@ -269,5 +269,9 @@ install-tool-scripts: install-destdir
 	cd tool-scripts/postprocess; \
 	${COPY} ${tool-postprocess} ${TOOLDIR}/postprocess
 
+install-build-artifacts:
+	${COPY} SHA1 ${DESTDIR}
+	if [ -f SEQNO ] ;then ${COPY} SEQNO ${DESTDIR} ;fi
```

Comment:
Aren't these provided in the tar ball at lines 56 and 57 of the `rpm/Makefile`?

Reply:
That's what builds them in the local tree, but they have to be copied into the tar ball that is used to build the srpm: otherwise, they are not going to be available to install.

Reply:
Removed the need for a special target because we just now copy the entire `agent` tree in place since the "install" used for constructing the source tar ball did the heavy lifting already.